tasks/flook.yml

```
@@ -0,0 +1,45 @@
+- name: Install flook dependencies
+  become: true
+  apt:
+    name:
+    - libreadline-dev
+
+- name: Get source for flook
+  get_url:
+    url: "{{ flook_url }}"
+    dest: "{{ flook_code_folder }}/{{ flook_src_archive }}"
+  register: flook_download
```

add some retries in case of bad connections

```suggestion
  register: flook_download
  until: not flook_download.failed
  retries: 3
  delay: 10
```